pom.xml

```diff
@@ -69,7 +69,7 @@ under the License.
   </distributionManagement>
 
   <properties>
-    <doxiaVersion>1.8</doxiaVersion>
+    <doxiaVersion>1.9-SNAPSHOT</doxiaVersion>
```

Comment:
This is a problem because we don't use snapshots in dependencies. This isn't stable/reliable. One needs to release Doxia 1.9 first.

Reply:
Understood.